packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -295,7 +299,7 @@ class BottomSheet extends Component {
 	onScroll( { nativeEvent } ) {
 		if ( this.isCloseToTop( nativeEvent ) ) {
 			this.setState( { bounces: false } );
-		} else if ( this.isCloseToBottom( nativeEvent ) ) {
+		} else {
```

Comment:
I found out an issue regarding this part when changing tabs. **I followed the following steps to reproduce it:**
1. Scroll down the blocks tab
1. Navigate to the reusable tab
1. Navigate back to the blocks tab
1. Observe that when scrolling up instead of scrolling the content the modal is being closed.

Curiously, the bounces value is used to control whether the modal should enable the swipe interaction that closes it so I had to modify it ([code reference](https://github.com/WordPress/gutenberg/blob/2a938548e8d2fed8a25b30db2b8675236089f176/packages/components/src/mobile/bottom-sheet/index.native.js#L410-L413)).

In any case, I think `bounces` should be enabled if it's not close to the top not only if it's closed to the bottom, I'm not sure what was the original purpose of this part but as far as I checked, this didn't introduce any regression.